globals.h

```diff
@@ -1,5 +1,5 @@
 /**
-Copyright 2018 JasmineGraph Team
```

Comment:
Why do we change the name of the `PlacesToNodeMapper.h` to `globals.h` ? Don't we need `PlacesToNodeMapper.h` to exist?

Reply:
` PlacesToNodeMapper.h` is removed (not replaced by globals.h). It was used in native mode only.